pkg/messages/messages.go

```diff
@@ -42,8 +44,13 @@ func (msg Message) Error() string {
 
 // Info ...
 func (msg Message) Info() string {
+	//If the BackendError is from library
 	if msg.BackendError != "" {
-		return fmt.Sprintf("{RequestID: %s, Code: %s, Description: %s, BackendError: %s, Action: %s}", msg.RequestID, msg.Code, msg.Description, msg.BackendError, msg.Action)
+		return fmt.Sprintf("{RequestID: %s, BackendError: %s, Action: %s}", msg.RequestID, msg.BackendError, msg.Action)
+	}
+	//If the CSIError is not empty
+	if msg.CSIError != "" {
+		return fmt.Sprintf("{RequestID: %s, Code: %s, Description: %s, Error: %s, Action: %s}", msg.RequestID, msg.Code, msg.Description, msg.CSIError, msg.Action)
 	}
 	return fmt.Sprintf("{RequestID: %s, Code: %s, Description: %s, Action: %s}", msg.RequestID, msg.Code, msg.Description, msg.Action)
```

Comment:
When this will be used?

Reply:
This will be used for all existing errors we populated from cSI Driver. Check the logs from below existing CSI and new CSI error

https://github.ibm.com/alchemy-containers/armada-storage/issues/6171#issuecomment-106016766

Reply:
Added documentation with example